CRM/EFT/BAO/EFT.php

```diff
@@ -84,7 +84,7 @@ public static function addChapterFund($chapter, $fund, $entityId, $entityTable,
         ];
         self::saveChapterFund($itemParams);
 
-        if ($isPriceSet && $financialItem) {
```

Comment:
I'm not sure why it would matter if it was a price set or not here given even in a non price set situation there should be financial trxn records

Reply:
This seems like a reasonable catch, @Edzelopez , but I am concerned about the underlying issues. We shouldn't be overloading a variable designed for one purpose like $isPriceSet to use it here and there for a different purpose. Thanks for getting this refactored out.